force-app/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -1,10 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Profile xmlns="http://soap.sforce.com/2006/04/metadata">
-    <applicationVisibilities>
```

Comment:
dont delete this because we want to still be able to view our custom app as admins